CHANGELOG.md

```diff
@@ -15,7 +15,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
     _Security_ in case of vulnerabilities.
  -->
 
-## [Unreleased](https://github.com/o1-labs/o1js/compare/f15293a69...HEAD)
+## [Unreleased](https://github.com/o1-labs/o1js/compare/7e9394...HEAD)
 
 ## [2.0.0](https://github.com/o1-labs/o1js/compare/f15293a69...HEAD)
```

Comment:
[NIT]

I think this compare should use `7e9394` as the base, rather than `f15293a69` which comes from v1.9.0

Reply:
Yep! Still need to shuffle commits around, I'll pick the final commit in a separate release PR